### PR TITLE
Update heat_units.md

### PR DESCRIPTION
**Pull request by LaplacetteCM**
This is the updated wiki for heat_units.py

### CODE REVIEW

File: SC_redesign/docs_SC/model_docs/heat_units.md [hunk 1]
```diff
@@ -1,80 +1,63 @@
-### __Heat units__
+### __Heat units__ 
 
-**Heat units** are calculated on a daily basis by the `HeatUnits`class.
-Each crop has its own temperature range for growth (i.e., minimum, optimum, and maximum) and`
-HeatUnits` holds for each particular crop all the methods that determine the **absorption** 
-and **accumulation** of heat units. 
+The `HeatUnits`class manages the accumulation of heat units for a crop on a daily basis. Crops have its own temperature range for growth (*i.e.*, minimum, optimum, and maximum) and `HeatUnits` holds for each particular crop all the methods that determine the absorption and accumulation of heat units.
```
Comment (morrowcj) [inline SC_redesign/docs_SC/model_docs/heat_units.md]:
```suggestion
The `HeatUnits`class manages the accumulation of heat units for a crop on a daily basis. Crops have their own temperature range for growth (*i.e.*, minimum, optimum, and maximum) and `HeatUnits` holds, for each particular crop, all the methods that determine the absorption and accumulation of heat units. 

```

**Review by morrowcj — APPROVED** [review]
I like these updates, Celina. I just have one comment suggesting grammar changes.

**Comment by LaplacetteCM** [comment]
> I like these updates, Celina. I just have one comment suggesting grammar changes.

Corrected! Thanks

File: SC_redesign/docs_SC/model_docs/heat_units.md [hunk 2]
```diff
@@ -1,80 +1,64 @@
...
-
-2. **Alternative method**: `heat_unit_temperature` is used instead of `air_temperature`. 
-Heat units accumulated are defined using the `heat_unit_temperature` resulting from the 
-comparison between minimum and maximum daily temperatures of crop and air:
-
-**Minimum heat unit temperature** calculates the minimum heat unit temperature on a given
-day using:
-```math
-  $$ T_{\text{min}} = 
+1. **Main method**: the mean air temperature (`mean_air_temperature`) is used as an input, and every degree above crop's minimum temperature for growth is accumulated as heat units. With  this method, where minimum temperature is dependant on the selected crop, heat units accumulated are always equal to the average air temperature (when above the minimum threshold).
```
Comment (Isgotkowitz) [inline SC_redesign/docs_SC/model_docs/heat_units.md]:
```suggestion
1. **Main method**: the mean air temperature (`mean_air_temperature`) is used as an input, and every degree above crop's minimum temperature for growth is accumulated as heat units. With this method, where minimum temperature is dependant on the selected crop, heat units accumulated are always equal to the average air temperature (when above the minimum threshold). 

```

**Review by Isgotkowitz — APPROVED** [review]
Looks good, only found one extra space that could be removed

**Pull request merged**